extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/helper/SparkSQLLineageParseHelper.scala

```diff
@@ -327,6 +336,19 @@ trait LineageParser {
           joinColumnsLineage(parentColumnsLineage, getSelectColumnLineage(p.aggregateExpressions))
         p.children.map(extractColumnsLineage(_, nextColumnsLineage)).reduce(mergeColumnsLineage)
 
+      case p: Expand =>
+        val childColumnsLineage = ListMap(p.output.collect {
+          case attr
+              if p.references.find(
+                _.name == attr.name.split('.').last.stripPrefix("`").stripSuffix("`")).nonEmpty =>
```

Comment:
What's the problem with `p.references.find(_.name == attr.name)` ? the name should have no qualifier

Reply:
Like this` Aggregate [a#0, spark_catalog.default.t2.`b`#10, spark_catalog.default.t2.`c`#11, gid#9]`, name be resovled with namspace

Reply:
I see. It should be a kind of bug of Spark... it's not a pretty name

Reply:
There was a problem with the way the association was based on the table name, remove this logic